cljfmt/test/cljfmt/test_util/common.cljc

```diff
@@ -0,0 +1,16 @@
+(ns cljfmt.test-util.common
+  (:require [clojure.string :as str]
+            [cljfmt.core :refer [reformat-string]]))
+
+(defn assert-reformats-to
+  ([msg in-lines expected-lines]
+   (assert-reformats-to msg in-lines {} expected-lines))
+  ([msg in-lines options expected-lines]
+   (let [input (str/join "\n" in-lines)
+         actual (reformat-string input options)
+         preserve-trailing-empty-strings -1
```

Comment:
I don't think we necessarily need this, since someone can just look up what `-1` means in this context.

Reply:
I didn't see it clearly documented, but now see it in user examples. Will comply.